woocommerce/google-listings-and-ads
# Use `woo-tracking-jsdoc` from external package

:warning: This PR requires https://github.com/woocommerce/grow/pull/13

### Changes proposed in this Pull Request:

Replace  `woo-tracking-jsdoc` with its external npm package


### Screenshots:




### Detailed test instructions:


1. Remove a few lines from https://github.com/woocommerce/google-listings-and-ads/blob/develop/src/Tracking/README.md?plain=1#L76-L90 to see that the result is generated
2. ```sh
   npm i
   npm run doc:tracking
   ```
3. Check that `Tracking/README.md` is generated correctly

### Additional details:

- Personally, I'm not very convinced that the bundling of jsdoc & co. done in https://github.com/woocommerce/google-listings-and-ads/compare/1e85b2a90d73d9fa22cdaea93f3ff046629452d2...55ebac9ec7c62429931f52474146b469e17b9ffb is the best solution, but it seems to me like an acceptable solution to limit the number of dependencies shared across multiple extensions.

### More to be done:

- [x] update branch name once https://github.com/woocommerce/grow/pull/6 gets merged


### Changelog entry
